docs/src/api/class-genericassertions.md

```diff
@@ -7,9 +7,9 @@ The [GenericAssertions] class provides assertion methods that can be used to mak
 ```js
 import { test, expect } from '@playwright/test';
 
-test('assert a value', async ({ page }) => {
+test('assert a value', ({ page }) => {
```

Comment:
```suggestion
test('assert a value', async ({ page }) => {
```

I'd keep that, since our customers have always async tests.

Reply:
I think adding `async` to this exact example would cause a linting issue on customer setups.

Reply:
The test should be async, I don't think lint should care about async functions not using await.

Reply:
Okay.  Force pushed